### PR TITLE
[BE][libpay] #WCN-2124: Change metadata sent with Stripe Charge

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by artempWix** [comment]
ticket number in branch name / commit message is invalid
it's related to Jira WCN-2124  - not WCN-2134

File: libpay-stripe/src/test/scala/com/wix/pay/stripe/StripeAdditionalInfoMapperTest.scala [hunk 1]
```diff
@@ -1,31 +0,0 @@
-package com.wix.pay.stripe
```
Comment (cponomaryov) [inline libpay-stripe/src/test/scala/com/wix/pay/stripe/StripeAdditionalInfoMapperTest.scala]: Why did you delete a test and didn't add the new one?

**Review by cponomaryov — CHANGES_REQUESTED** [review]

File: libpay-stripe/src/test/scala/com/wix/pay/stripe/StripeAdditionalInfoMapperTest.scala (hunk 1 above)
Comment (artempWix) [reply libpay-stripe/src/test/scala/com/wix/pay/stripe/StripeAdditionalInfoMapperTest.scala]: i've lost them after a set of rebases / ammends
restored from local history + added refactoring
thanks!

**Review by cponomaryov — APPROVED** [review]
